src/bp/nlu-core/model-manager.ts

```diff
@@ -0,0 +1,53 @@
+import * as sdk from 'botpress/sdk'
+import _ from 'lodash'
+
+import { TrainInput, TrainOutput } from './training-pipeline'
+import { EntityCache } from './typings'
+
+export type Model = Omit<sdk.NLU.Model, 'data'> & {
```

Comment:
This line is a bit confusing it's hard to tell what's the diff between a Model and a NLU.Model if you don't know the inner details of data interface... would SerializableModel be a better naming ?

Reply:
In the model-manager, the `Model` type is actually a type where `input` and `output` have been built back from serialization. The correct name would actually be something like `BuiltModel`, but this feels even weirder. 

Yeah, kind of blocked on this one...

Reply:
That's a `PredictableModel`

Reply:
yes